viz/Apex-Radar/src/common.ts

```diff
@@ -0,0 +1,312 @@
+import {
+  ObjectFormat,
+  RowEntry,
+  ObjectRow,
+  FieldsByConfigId,
+  StyleById,
+} from '@google/dscc';
+import ApexCharts from 'apexcharts';
+
+interface FontInfo {
+  family: string;
+  size: number;
+  color: string;
+}
+interface Chart {
+  chart: ApexCharts;
+  element: Element;
+}
+let myChart: Chart = {
+  chart: undefined,
+  element: undefined,
+};
+const CHARTNAME = 'myChart';
+
+export function drawViz(data: ObjectFormat): void {
+  const newChart: boolean = document.getElementById(CHARTNAME) == undefined;
+  const tables = populateTables(data.tables.DEFAULT, data.fields);
+  const styling = populateStyle(data.style, tables.labels.length);
+
+  const options = {
+    series: tables.series,
+    labels: tables.labels,
+    tooltip: styling.tooltip,
+    chart: styling.chart,
+    fill: styling.fill,
+    markers: styling.markers,
+    dataLabels: styling.datalabels,
+    plotOptions: styling.plotoptions,
+    xaxis: styling.xaxis,
+    yaxis: styling.yaxis,
+    stroke: styling.stroke,
+    legend: styling.legend,
+  };
+  if (newChart) {
+    myChart.element = document.createElement('div');
+    myChart.element.id = CHARTNAME;
+    document.body.appendChild(myChart.element);
+    myChart.chart = new ApexCharts(myChart.element, options);
+    myChart.chart.render();
```

Comment:
(Related to above comment about myChart)

```suggestion
    const newChartElement = document.createElement('div');
    myChartElement.id = CHARTNAME;
    document.body.appendChild(myChartElement);
    const newApexChart = new ApexCharts(myChartElement, options);
    newApexChart.render();
    myChart = { element: myChartElement, chart: newApexChart };
```

The suggestion is to fix the same issue I mentioned above - having a partially filled struct doesn't help much.

(Both suggestions aren't as important if you decide to keep `strictNullChecks` off for this viz)